### PR TITLE
Updated link to routing

### PR DESCRIPTION
**Pull request by neugartf**

### CODE REVIEW

File: documentation/tree-structure-101/happy-little-trees.md [hunk 1]
```diff
@@ -48,8 +48,6 @@ Which means that we now have three levels in the tree actually:
 While extracting screens as standalone ```Nodes``` rather explains itself, we should go further. 
  Going deeper, sections of the screen can be thought of as standalone components implementing a concern: extract them as children and delegate this responsibility to them.
 
-[image]()
```
Comment (zsoltk) [inline documentation/tree-structure-101/happy-little-trees.md:51]: I'd rather fix the image myself then remove this and forget about it :)

**Review by zsoltk — CHANGES_REQUESTED** [review]

File: documentation/tree-structure-101/happy-little-trees.md (hunk 1 above)
Comment (zsoltk) [reply documentation/tree-structure-101/happy-little-trees.md:51]: TBC: Not expecting you to fix it, I'll do that – just don't remove it please.

Comment (neugartf) [reply documentation/tree-structure-101/happy-little-trees.md:51]: Sure thing!

**Comment by zsoltk** [comment]
Thanks @neugartf!

**Pull request merged**